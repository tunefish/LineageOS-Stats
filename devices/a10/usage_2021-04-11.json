{
  "10.3": 1,
  "16.0": 69,
  "17.1": 876,
  "18.1": 429,
  "none": 412
}