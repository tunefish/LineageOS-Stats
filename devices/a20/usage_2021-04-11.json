{
  "17.1": 2433,
  "18.1": 199,
  "none": 28
}