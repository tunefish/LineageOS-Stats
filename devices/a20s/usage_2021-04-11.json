{
  "17.1": 2493,
  "none": 1
}