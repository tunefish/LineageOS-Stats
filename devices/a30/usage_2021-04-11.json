{
  "17.1": 37,
  "18.0": 744,
  "18.1": 403,
  "none": 4
}