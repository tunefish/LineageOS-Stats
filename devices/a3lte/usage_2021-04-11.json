{
  "15.1": 733,
  "16.0": 264,
  "17.1": 897
}