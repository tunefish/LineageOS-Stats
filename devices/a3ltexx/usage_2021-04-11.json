{
  "14.1": 1127
}