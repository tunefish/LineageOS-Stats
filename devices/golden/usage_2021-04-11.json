{
  "13.0": 1281,
  "14.1": 4467,
  "15.1": 5,
  "17.1": 8,
  "none": 1
}